iptables/rules.go

```diff
@@ -166,3 +166,20 @@ func (c *Chain) RuleHashes(features *Features) []string {
 	}
 	return hashes
 }
+
+func (c *Chain) IPSetIDs() (ipSetIDs []string) {
+	if c == nil {
+		return nil
+	}
+	for _, rule := range c.Rules {
+		for _, matchString := range []string(rule.Match) {
```

Comment:
I think this for loop should be in a method on the `MatchCriteria` type so that the string parsing is alongside the code that generates the strings.  Then this loop can be simply:
```
for _, rule := range c.Rules {
  ipSetIDs = append(ipSetIDs, r.Match.IPSetIDs())
}
```